cmd/kube-apiserver/app/server.go

```diff
@@ -120,8 +119,7 @@ func NewAPIServer() *APIServer {
 		SecurePort:             6443,
 		APIRate:                10.0,
 		APIBurst:               200,
-		APIPrefix:              "/api",
-		ExpAPIPrefix:           "/experimental",
+		APIPrefix:              "/kube-api",
```

Comment:
As we discussed, creation of a new KubeAPIPrefix flag should be moved to a separate PR.